03-Mains/Meridional_heat_transport.ipynb

```diff
@@ -6,45 +6,49 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    
</pre><p>what's all these commented lines?</p><p>let's delete if not needed!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>